Assets/Scripts/AI/RacerManager.cs

```diff
@@ -9,6 +9,7 @@ public class RacerManager : MonoBehaviour
     private List<Rigidbody> racerRigidbodies = new List<Rigidbody>();
     private List<PositionTracker> positionTrackers = new List<PositionTracker>();
     private int numFinishedRacers;
+    private bool FirstHit = true;
```

Comment:
again minor detail but according to our coding style a variable name should start with a non-capitalized letter